src/components/accessible-autocomplete/accessible-autocomplete.browser.test.js

```diff
@@ -0,0 +1,345 @@
+      ],
+    }));
+
+    const element = await page.$('#location');
+    const tagName = await element.evaluate((el) => el.tagName.toLowerCase());
+    const borderColor = await element.evaluate((el) => getComputedStyle(el).getPropertyValue('border-color'));
+
+    expect(tagName).not.toBe('select'); // or select element was not enhanced to be an autocomplete component
+    expect(borderColor).toBe('rgb(212, 53, 28)');
+  });
+
+  it.failing('should not retain the previous selection if an invalid option is entered', async () => {
+    await render(page, withGovukSelect({
+      id: 'location',
+      name: 'location',
+      attributes: {
+        'data-module': 'hmrc-accessible-autocomplete',
+        'data-auto-select': 'false',
+        // this is the default, but included to be explicit with test state
+        // auto select would mean that you don't have to enter exactly matching
+        // text to select on blur
+      },
+      label: {
+        text: 'Choose location',
+      },
+      items: [
+        {
+          value: ' ',
+          text: 'Choose location',
+        },
+        {
+          value: 'london',
+          text: 'London',
+        },
+        {
+          value: 'southwest',
+          text: 'South West',
+        },
+      ],
+    }));
+
+    await expect(page).toFill('#location', 'Lon');
+    await acceptFirstSuggestionFor('#location');
+    expect(await page.$eval('select', (select) => select.value)).toBe('london');
+    await expect(page).toFill('#location', 'South');
+    await page.$eval('#location', (input) => input.blur());
+    expect(await page.$eval('select', (select) => select.value)).toBe('');
+  });
+
+  it.failing('should select any option with exactly matching text on blur, even if it was not chosen from the suggestions', async () => {
+    await render(page, withGovukSelect({
+      id: 'location',
+      name: 'location',
+      attributes: {
+        'data-module': 'hmrc-accessible-autocomplete',
+        'data-auto-select': 'false',
+        // this is the default, but included to be explicit with test state
+        // auto select would mean that you don't have to enter exactly matching
+        // text to select on blur
+      },
+      label: {
+        text: 'Choose location',
+      },
+      items: [
+        {
+          value: ' ',
+          text: 'Choose location',
+        },
+        {
+          value: 'london',
+          text: 'London',
+        },
+        {
+          value: 'southwest',
+          text: 'South West',
+        },
+      ],
+    }));
+
+    await expect(page).toFill('#location', 'Lon');
+    await acceptFirstSuggestionFor('#location');
+    expect(await page.$eval('select', (select) => select.value)).toBe('london');
+    await expect(page).toFill('#location', 'South West');
+    await page.$eval('#location', (input) => input.blur());
+    expect(await page.$eval('select', (select) => select.value)).toBe('southwest');
+  });
```

Comment:
this is the bit that I thought was additional behaviour, but actually I think adam's patch will be achieving this, it's just that my approach that I proof of concepted wouldn't be - onConfirm will get "undefined" in this case, so you have to in this case lookup the current value of the autocomplete input instead and use that to mark an option as selected